source/OutfitterPanel.cpp

```diff
@@ -189,13 +200,19 @@ void OutfitterPanel::DrawItem(const string &name, const Point &point)
 		if(maxCount)
 		{
 			string label = "installed: " + to_string(minCount);
+			Color color = bright;
```

Comment:
```suggestion
			const Color &color = bright;
```

Reply:
It needs to be possible to update the value of this `Color` further down, so making it `const` won't work, and making it a non-const reference isn't allowed because `bright` is `const`, and even if it were allowed, we don't want to update the value of `bright`.

Reply:
ah right, nevermind then